pkg/api/types.go

```diff
@@ -1368,6 +1368,13 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+type Heartbeat struct {
+	// Time when this heartbeat was generated.
```

Comment:
I don't think I need. There's a lot of various stuff, that's completely unimportant for a hearbeat (name, uuid, etc.). Binding is a different case, as it represents a binary relation, and we store one dimension as ObjectMeta, and second as ObjectReference. Here we need only one reference - I don't really care if it's ObjectMeta, or ObjectReference, but ObjectReference is smaller, and have everything we need (or at least everything I believe we need).